### ISSUE TITLE
Issue with "t" function in /lib/fat_free_crm/i18n.rb

### ISSUE THREAD

**Issue by connectGBI** [fatfreecrm/fat_free_crm#1102]
Hello, 

I've had an issue with /lib/fat_free_crm/i18n.rb with the "t" function. I am extremely new to all of this, but I think i found a solution so it stops breaking. I was getting issues that said:

ArgumentError (wrong number of arguments (given 2, expected 1)):
  
Any  time "t" was called with an argument that was an array. The app seems to work now. 

Here's the code I used for the "t" function:

def t(*args)
  if args.size == 1
    super(args.first, default: args.first.to_s)
  elsif args.second.is_a?(Hash)
    if args.second.value?(Array)
      options = args.second.each_with_object({}) do |(k, v), h|
        h[k] = v.is_a?(Array) ? v.join(',') : v
      end
      super(args.first, **options)
    else
      super(*args)
    end 
  elsif args.second.is_a?(Integer)
    super(args.first, count: args.second)
  else
    super(args.first, value: args.second)
  end
end

Is this an alright fix? Is it going to cause me problems down the road?

**Comment by ferrisoxide** [comment]
Had the same issue trying to get the app working locally (see PR #1135). The fix I tried looks like this:

```ruby
    def t(*args)
      if args.size == 1
        super(args.first, default: args.first.to_s)
      elsif args.second.is_a?(Hash)
        super(args.first, **args.second)
      elsif args.second.is_a?(Integer)
        super(args.first, count: args.second)
      else
        super(args.first, value: args.second)
      end
    end
```

It _seems_ to work, but I'm also fairly new to FatFreeCRM - so I've not sure what's the best approach.

**Comment by ferrisoxide** [comment]
@connectGBI 

I've split my fix into a separate PR (see #1144). If you're keen, maybe see if that works for you.
